.ci/docker/Dockerfile_zeebe-hazelcast-exporter

```diff
@@ -0,0 +1,12 @@
+FROM camunda/zeebe:current-test
```

Comment:
What is current-test here?

Reply:
`current-test` is the version that is built in prepare test stage of the build pipeline. It is only available locally and never pushed to any registry. It is also the one used for the integration tests (and I guess go tests, too)